lib/renuo/bin-check/printer.rb

```diff
@@ -7,7 +7,9 @@ def print_output(results)
 
     # :reek:UtilityFunction
     def print_error_output(result)
-      $stderr.puts result.error_output
+      error_output = result.error_output
+      error_output = result.output if error_output == ''
```

Comment:
i'm not sure if this is really better, but this would be a shorter syntax but only work if the error_output is `nil` if not availabe:
```rb
error_output = result.error_output || error_output

```